Program.cs

```diff
@@ -15,252 +15,273 @@ public static void printsContentsRefrigerator(List<Refrigerator> allfrige)
+            allIShelfes.Add(new Shelf(1, 8, items1));
+            allIShelfes.Add(new Shelf(2, 7, items2));
+            allIShelfes.Add(new Shelf(3, 3, items3));
             //update shelf of all item that insert to friger
-            allItems[0]._shelfOfItem= allShelfes[0];
-            allItems[1]._shelfOfItem = allShelfes[0];
-            allItems[2]._shelfOfItem = allShelfes[0];
-            allItems[3]._shelfOfItem = allShelfes[1];
-            allItems[4]._shelfOfItem = allShelfes[1];
-
-            List<Shelf> t = new List<Shelf>();
-            t.Add(new Shelf(3, 20, items3));
-            t.Add(new Shelf(4, 40, items3));
-
-            List<Refrigerator> allfrige = new List<Refrigerator>();
-            allfrige.Add(new Refrigerator(Refrigerator.Model.LG_GR_X265, Refrigerator.Colors.Aquamarine, 4, allShelfes));
-            allfrige.Add(new Refrigerator(Refrigerator.Model.LG_GR_X265, Refrigerator.Colors.Aquamarine, 4, t));
+            allItems[0]._idShelf = allIShelfes[0].getId();
+            allItems[1]._idShelf = allIShelfes[0].getId();
+            allItems[2]._idShelf = allIShelfes[0].getId();
+            allItems[3]._idShelf = allIShelfes[1].getId();
+            allItems[4]._idShelf = allIShelfes[1].getId();
+            allItems[5]._idShelf = allIShelfes[2].getId();
+            allIFriger.Add(new Refrigerator(Refrigerator.Model.LG_GR_X265, Refrigerator.Colors.Aquamarine, 4, allIShelfes));
             #endregion
-
-            bool exit = false;
-            Console.WriteLine("Hello, you have arrived at the program \"The Refrigerator\"\r\nYou can choose one of the options");
-            while (!exit)
+        }
+        public static string ShowMenu()
+        {
+            Console.WriteLine("1. Printing the refrigerator and all its contents");
+            Console.WriteLine("2. Printing the space left in the fridge");
+            Console.WriteLine("3. Put an item in the fridge");
+            Console.WriteLine("4. Take an item out of the fridge");
+            Console.WriteLine("5. Throw away all expired products from the refrigerator");
+            Console.WriteLine("6. Please select the type of food and kosher of the item you wish to eat");
+            Console.WriteLine("7. Receiving all products sorted by expiration date");
+            Console.WriteLine("8. Getting all the shelves sorted according to the free space left on them");
+            Console.WriteLine("9. Getting all the refrigerators sorted according to the free space left on them");
+            Console.WriteLine("10. Preparing the fridge for shopping");
+            Console.WriteLine("100. Exit");
+            Console.WriteLine("");
+            string choice = Console.ReadLine();
+            return choice;
+        }
+        public static void ShowFunctionMenu(int numFunction, List<Item> allItems, List<Shelf> allIShelfes, List<Refrigerator> allIFriger)
+        {
+            switch (numFunction)
```

Comment:
change the switch case pattern in all of the code to make it more efficient, think on another way, to wrap it up.
text me if you dont find it.